mycroft_voice_satellite/speech/stt_factory.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from time import time, sleep
+import os
+import platform
+import posixpath
+import tempfile
+import requests
+from contextlib import suppress
+from glob import glob
+from os.path import dirname, exists, join, abspath, expanduser, isfile, isdir
+from shutil import rmtree
+from threading import Timer, Event, Thread
+from urllib.error import HTTPError
+
+from mycroft_voice_satellite.configuration import CONFIGURATION
+from ovos_utils.log import LOG
+from ovos_plugin_manager.stt import load_stt_plugin
+
+
+
+class STTFactory:
+    _MAPPINGS = {
+        "google": "ovos-stt-plugin-chromium",
+        "vosk": "ovos-stt-plugin-vosk"
+    }
```

Comment:
Doesn't hardcoding supported plugins defeat the purpose of using plugins?

Reply:
this is a just a mapping to provide backwards compat, it translates the "old strings" into the plugin identifier

but i do provide plugins for the default options in requirements.txt